config/sounds-web/url.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { options } = require('./common');
+
+module.exports = {
+  options,
+  baseUrl: process.env.URL,
+  paths: ['']
```

Comment:
```suggestion
  baseUrl: process.env.A11Y_SOUNDS_WEB_BASE_URL,
  paths: [A11Y_SOUNDS_WEB_PATH]
```
How about something like this? Keeps the same env naming conventions then.

Reply:
I decided to use URL instead because its shorter. The difference:
`URL='http://www.google.com' npm run test:a11y-url`
`A11Y_SOUNDS_WEB_BASE_URL='http://www.google.com' npm run test:a11y-url`
I feel like the second one is more effort to type, as that part can't be included in the sounds web scripts and has to be added each time.

What does `A11Y_SOUNDS_WEB_PATH` refer to?

Reply:
You could put `A11Y_SOUNDS_WEB_BASE_URL` in your `.bash_profile` or equivalent so you don't have to type it every time.
Then you can do:
A11Y_SOUNDS_WEB_PATH='/sounds/newpage' npm run test:a11y-url

Reply:
(Also, reason I suggest the longer variable name is so that it doesn't clash with variables in your environment already)

Reply:
Or maybe as a `--` param? https://github.com/npm/npm/pull/5518